pkg/parser/yamlparser.go

```diff
@@ -248,6 +248,22 @@ func (doc *YamlDocument) CouldBeOrbReference(orbReference string) (string, bool)
 	return splittedCommand[0], true
 }
 
+// Takes the name of anything that may be in an orb and returns if it is inside an orb that we can not use
+func (doc *YamlDocument) IsFromUnfetchableOrb(name string) bool {
+	splittedName := strings.Split(name, "/")
```

Comment:
[The past of "split" is "split"](https://simple.wiktionary.org/wiki/split#Verb) not "splitted"... 😬 

Do you want to rename this to `splitName`? Or maybe `nameComponents` (or even just `components`)?

Reply:
Thank you, fyi I think there are many other `splitted` in the codebase 😅 
But of course will change them when I encounter them

Reply:
😂 

No worries, typos happen. That's also what reviews are for anyway.

It's harder when it's in a public interface... 😅